src/create_torrent.cpp

```diff
@@ -455,6 +456,12 @@ namespace {
 			aux::throw_ex<system_error>(errors::invalid_piece_size);
 		}
 
+		// this is an unreasonably large piece size. Some clients don't support
+		// pieces this large.
+		if (piece_size > 256 * 1024 * 1024) {
```

Comment:
At least, I expect libtorrent to have the same limit for creating and using torrents.